apps/judicial-system/backend/src/app/modules/police/police.service.ts

```diff
@@ -37,8 +38,45 @@ import { PoliceCaseInfo } from './models/policeCaseInfo.model'
 export class PoliceService {
   private xRoadPath: string
   private agent: Agent
-
   private throttle = Promise.resolve({} as UploadPoliceCaseFileResponse)
+  private policeCaseFileStructure = z.object({
```

Comment:
Also, do you think we should only validate the things we are using from this API or validate the whole response, like we are doing here?

Reply:
I think we should only validate what we are using. I expect the response to change over time and have a lot of stuff we are not using. We should tolerate changes to unused parts of the data.